### PR TITLE
Added Challenges on mobile

### PR DESCRIPTION
**Pull request by MDGcode**
Please merge when you think its ready

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -46,6 +46,8 @@ body {
   min-height: 100vh;
 }
 
+
+
```
Comment (cristim67) [inline client/src/index.css]: delete this.

File: client/src/section/ChallengesMobile.tsx [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            className={`relative ${
+              expandedDiv === 1 ? "h-[50%]" : "h-[30%]"
+            } bg-white flex flex-col justify-center items-center rounded-xl transition-all duration-500`}
+          >
+            <img
+              className={`mb-4 scale-[90%] relative top-2 ${
+                expandedDiv === 1 ? "mt-10" : "mt-5"
+              }`}
+              src={Proba1}
+              alt=""
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: put a value here in alt tag.

File: client/src/section/ChallengesMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+                <h1 className="text-lg tracking-tighter text-left ml-5  text-black ">
+                  Scopul acestei probe consta in dezvoltarea unui sistem
+                  specializat pentru imbunatatirea, restaurarea si analiza
+                  imaginilor in contextul securitatii, cu utilizarea
+                  tehnologiilor de inteligenta artificiala
+                </h1>
+
+                <div className="text-black mt-10 ml-5 ">
+                  <div className="flex items-center mb-2">
+                    <img src={CheckIcon} alt="" className="mr-5" />
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: same here

File: client/src/section/ChallengesMobile.tsx [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+                    <h3 className="mr-6">Super Resolution</h3>
+                  </div>
+                  <div className="flex items-center mb-2">
+                    <img src={CheckIcon} alt="" className="mr-5" />
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: same here.

File: client/src/section/ChallengesMobile.tsx [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+                    <h3 className="mr-6">Denoising</h3>
+                  </div>
+                  <div className="flex items-center mb-2">
+                    <img src={CheckIcon} alt="" className="mr-5" />
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: same here.

File: client/src/section/ChallengesMobile.tsx [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+                  <Fragment></Fragment>
+                  <span className="text-white">{">"}</span>
+                </span>
+              </button>
+              <img
+                src={PremiiConnect}
+                className="absolute -z-10 scale-[200%]"
+              />
+            </div>
+          </div> */}
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: delete this.

File: client/src/views/Home.view.tsx [hunk 7]
```diff
@@ -48,22 +49,26 @@ const Home = () => {
...
           <MainText />
-            <a>
-              <button className="bg-[#FFC947] rounded-md  -mt-2 mb-10 text-black w-60 flex items-center absolute mt-10 h-12 left-1/2 transform -translate-x-1/2"
-                onClick={() =>
-                  (window.open('https://cyber-edu.co/register?token=40a121ba7d8ac969b5d08e041632f81ad4a599d11f054d4d1812ac94f480c4e34b4a9548de58bdf3', '_blank'))
-                }
-              >
-                <span className="mr-2 scale-125 absolute left-10 font-bold">
-                  Inscrie-te aici
+          <a>
```
Comment (cristim67) [inline client/src/views/Home.view.tsx]: you have a `<a>` tag without href.

File: client/src/views/Home.view.tsx [hunk 8]
```diff
@@ -144,14 +146,17 @@ const Home = () => {
               className=" absolute z-10 -translate-y-40 scale-100"
               alt="Location"
             />
+            <div className="  my-80"></div>
             <img
               src={Date}
-              className=" absolute z-20 translate-y-5 scale-100"
+              className=" absolute z-20 translate-y-10 scale-[100%]"
```
Comment (cristim67) [inline client/src/views/Home.view.tsx]: scale-100 or scale-[100%] has no effect here

File: client/src/section/ChallengesMobile.tsx [hunk 9]
```diff
@@ -0,0 +1,307 @@
+import { Fragment, useState } from "react";
+
+import ChallengesText from "../assets/Challenges.svg";
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: delete this if you don t use it.

File: client/src/section/ChallengesMobile.tsx [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+                <span className=" text-sm font-bold ">Inscrie-te aici</span>
+                <span className="inline-block bg-black text-white rounded-full h-6 w-6  justify-center items-center ml-5">
+                  {" "}
+                  <Fragment></Fragment>
+                  <span className="text-white">{">"}</span>
+                </span>
+              </button>
+              <img
+                src={PremiiML}
+                className="absolute flex justify-center -right-5 -z-10 scale-[220%]"
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: add alt tag

File: client/src/section/ChallengesMobile.tsx [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+                <span className="inline-block bg-black text-white rounded-full h-6 w-6  justify-center items-center ml-5">
+                  {" "}
+                  <Fragment></Fragment>
+                  <span className="text-white">{">"}</span>
+                </span>
+              </button>
+              <img
+                src={PremiiCapture}
+                className="absolute flex justify-center -right-0 -z-10 scale-[220%]"
+              />
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: same here.

File: client/src/section/ChallengesMobile.tsx [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+                <span className=" text-sm font-bold ">Inscrie-te aici</span>
+                <span className="inline-block bg-black text-white rounded-full h-6 w-6  justify-center items-center ml-5">
+                  {" "}
+                  <Fragment></Fragment>
+                  <span className="text-white">{">"}</span>
+                </span>
+              </button>
+              <img
+                src={PremiiConnect}
+                className="absolute flex justify-center right-5 -z-10 scale-[220%]"
```
Comment (cristim67) [inline client/src/section/ChallengesMobile.tsx]: same here.

File: client/src/views/Home.view.tsx [hunk 13]
```diff
@@ -48,22 +49,26 @@ const Home = () => {
...
-              <button className="bg-[#FFC947] rounded-md  -mt-2 mb-10 text-black w-60 flex items-center absolute mt-10 h-12 left-1/2 transform -translate-x-1/2"
-                onClick={() =>
-                  (window.open('https://cyber-edu.co/register?token=40a121ba7d8ac969b5d08e041632f81ad4a599d11f054d4d1812ac94f480c4e34b4a9548de58bdf3', '_blank'))
-                }
-              >
-                <span className="mr-2 scale-125 absolute left-10 font-bold">
-                  Inscrie-te aici
+          <a>
+            <button
+              className="bg-[#FFC947] rounded-md  -mt-2 mb-10 text-black w-60 flex items-center absolute mt-10 h-12 left-1/2 transform -translate-x-1/2"
```
Comment (cristim67) [inline client/src/views/Home.view.tsx]: here you give two values ​​for the same attribute, rewriting the first value
`-mt-2` and `mt-10` applies same css properties.

File: client/src/views/Home.view.tsx [hunk 14]
```diff
@@ -48,22 +49,26 @@ const Home = () => {
...
+                  "https://cyber-edu.co/register?token=40a121ba7d8ac969b5d08e041632f81ad4a599d11f054d4d1812ac94f480c4e34b4a9548de58bdf3",
+                  "_blank"
+                )
+              }
+            >
+              <span className="mr-2 scale-125 absolute left-10 font-bold">
+                Inscrie-te aici
+              </span>
+              <div className="absolute right-2.5">
+                <span className="inline-block   bg-black text-white rounded-full h-6 w-6 flex justify-center items-center">
```
Comment (cristim67) [inline client/src/views/Home.view.tsx]: same here `flex` and `inline-block`

**Review by cristim67 — APPROVED** [review]

File: client/src/views/Home.view.tsx (hunk 7 above)
Comment (MDGcode) [reply client/src/views/Home.view.tsx]: The link will be added later

**Pull request merged**